src/Symfony/Component/Mailer/Bridge/Sendgrid/composer.json

```
@@ -17,13 +17,17 @@
     ],
     "require": {
         "php": ">=8.1",
+        "psr/event-dispatcher": "^1",
+        "starkbank/ecdsa": "^0.*",
```

A 0.* version is not stable from a semver POV

---

I see a version 2.x on Packagist, maybe we can use it?
Also, it should not be part of the require section as this is optional, so you can move it to request-dev and throw an exception if not available in the code.